.github/workflows/nightly-dev-doc-build.yml

```diff
@@ -20,18 +20,10 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      # used for documentation deployment
```

Comment:
This is no longer needed then?

Reply:
We are not using this kind of token in our workflows anymore.